medusa/indexers/tvdbv2/tvdbv2_api.py

```diff
@@ -139,8 +141,12 @@ def _show_search(self, show, request_language='en'):
         """
         try:
             results = self.search_api.search_series_get(name=show, accept_language=request_language)
+        except ApiException as e:
+            raise IndexerShowNotFound(
+                'Show search failed in getting a result with reason: %s (%s)' % (e.reason, e.status)
+            )
         except Exception as e:
-            raise IndexerException('Show search failed in getting a result with error: %r', e)
+            raise IndexerException('Show search failed in getting a result with error: %r' % e)
```

Comment:
I think it's still possible to get exceptions raised that are not catched with IndexerException. Like SSL or other connection errors. So at this stage, i'd like to normalize it, and use the IndexerExceptions.